vumi_message_store/tests/test_riak_backend.py

```diff
@@ -22,15 +26,27 @@ def vumi_date(timestamp):
     return datetime.strftime(timestamp, VUMI_DATE_FORMAT)
 
 
-class TestMessageStoreRiakBackend(VumiTestCase):
+class RiakBackendTestMixin(object):
 
-    def setUp(self):
-        self.persistence_helper = self.add_helper(
-            PersistenceHelper(use_riak=True))
-        self.manager = self.persistence_helper.get_riak_manager()
-        self.backend = MessageStoreRiakBackend(self.manager)
+    def set_up_tests(self):
+        """
+        This should be called from .setUp().
+        """
+        self.backend = self.get_backend()
         self.msg_helper = self.add_helper(MessageHelper())
 
+    def get_backend(self):
+        """
+        Construct a backend object.
+        """
+        raise NotImplementedError(".get_backend() must be implemented.")
+
+    def get_model_proxy(self, modelcls):
+        """
+        Construct a model proxy for the given model class.
+        """
+        raise NotImplementedError(".get_model_proxy() must be implemented.")
```

Comment:
We don't seem to be using `get_model_proxy`. Are we keeping it around for something in future?